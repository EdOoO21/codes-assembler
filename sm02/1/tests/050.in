752 -455 -255 305 824 -504 418 -31 570 987 -325 -826 102 -351 -541 385 140 -826 905 -687 -289 690 -799 -289 -346 -545 -613 -860 991 770 -327 236 829 244 -122 300 -818 874 899 971 -869 419 236 689 -316 625 414 317 705 -119 -989 -535 -309 -455 635 904 -111 433 -784 -589 865 -94 -445
