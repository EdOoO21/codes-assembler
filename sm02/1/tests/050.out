-445
-94
865
-589
-784
433
-111
904
635
-455
-309
-535
-989
-119
705
317
414
625
-316
689
236
419
-869
971
899
874
-818
300
-122
244
829
236
-327
770
991
-860
-613
-545
-346
-289
-799
690
-289
-687
905
-826
140
385
-541
-351
102
-826
-325
987
570
-31
418
-504
824
305
-255
-455
752
